tests/circuit_graph/test_circuit_graph_hash.py

```diff
@@ -23,23 +23,22 @@
 from pennylane.wires import Wires
 
 
-@pytest.mark.usefixtures("use_legacy_opmath")
+@pytest.mark.usefixtures("legacy_opmath_only")
```

Comment:
Do we not test this with new opmath?

Reply:
Mistake from a "Find and replace all" 😅 . Just checked and the tests pass with both new and legacy op math, so I've removed the decorator.